packages/utils/src/set.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
```

Comment:
Probably worth it to add tests for these utilities, especially since they're pretty simple. A test suite doesn't exist for utils right now, but I think they could simply be added to core tests. If it makes things easier, the utils folder could even simply be moved there for now.